### PR TITLE
Fixup HPA integration tests

### PR DESCRIPTION
**Pull request by gionn**

### CODE REVIEW

File: test/autoscaling-hpa-test-values.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ alfresco-repository:
     maxReplicas: 2
     behavior:
       scaleUp:
-        stabilizationWindowSeconds: 10
+        stabilizationWindowSeconds: 0
```
Comment (pmacius) [inline test/autoscaling-hpa-test-values.yaml:8]: This is to make sure we dont delay the scaling process?

File: .github/workflows/helm-enterprise.yml [hunk 2]
```diff
@@ -117,8 +117,7 @@ jobs:
         run: |
           if [ "${{ matrix.values }}" = "values.yaml" -o "${{ matrix.values }}" = "pre-release_values.yaml" ]; then
             echo "Enabling clustered tests via auto-scaling: 2 replicas max, 10s stabilization window"
-            echo "helm_install_params=--values test/autoscaling-hpa-test-values.yaml \
-              --set dtas.additionalArgs[0]='-k not test_renditions'" >> "$GITHUB_OUTPUT"
```
Comment (pmacius) [inline .github/workflows/helm-enterprise.yml:121]: We removing this to make sure its overloaded?

**Review by pmacius — APPROVED** [review]
Making sure I understand the changes

File: test/autoscaling-hpa-test-values.yaml (hunk 1 above)
Comment (gionn) [reply test/autoscaling-hpa-test-values.yaml:8]: yes, seems more sensibile for testing even if...

File: .github/workflows/helm-enterprise.yml (hunk 2 above)
Comment (gionn) [reply .github/workflows/helm-enterprise.yml:121]: I think it was a mistake to have this on master, maybe was a leftover from tests

the renditions skip was really necessary for acs 7.0 - and we dropped support for it in the meanwhile

**Review by pmacius — APPROVED** [review]

**Pull request merged**